pkg/oauth/providers/github.go

```diff
@@ -38,6 +38,7 @@ func getGitHubUser(c *gin.Context, token *oauth2.Token) (*models.UserCredentials
 		LoggedIn:     true,
 		SocialAuthID: strconv.FormatInt(*githubUser.ID, 10),
 		CreatedAt:    &currTime,
+		Photo:        githubUser.GetAvatarURL(),
```

Comment:
What will happen in case of Google?

Reply:
Fog google, I'm already setting the photo url from the Oauth provider to the userModel which is returned to a function which does a Create-Or-Get User from mongo.

Reply:
I had missed adding it for GitHub earlier.

```go
        SocialAuthID:    gUser.ID,    
        LoggedIn:        true,    
        Photo:           gUser.Picture,    
        OnBoardingState: models.BoardingStateEmailVerified,    
```